woof-code/rootfs-petbuilds/gtkdialog/globals.patch

```diff
@@ -0,0 +1,80 @@
+diff -rup gtkdialog-0.8.4a-orig/src/gtkdialog.c gtkdialog-0.8.4a/src/gtkdialog.c
```

Comment:
@01micko are gtkdialog, Xdialog, etc' up-to-date against Slacko 7.0 here?

Reply:
Yes.

gtkdialog - https://distro.ibiblio.org/puppylinux/sources/g/gtkdialog-0.8.4a.tar.xz
Xdialog - https://chiselapp.com/user/jamesbond/repository/xdialog/timeline

I'll upload Xdialog tarball to ibiblio

Reply:
Thanks! Will update.